prisoners_dilemma_sig
iter : 0
iter : 100
iter : 200
iter : 300
iter : 400
iter : 500
iter : 600
iter : 700
iter : 800
iter : 900
iter : 1000
iter : 1100
iter : 1200
iter : 1300
iter : 1400
iter : 1500
iter : 1600
iter : 1700
iter : 1800
iter : 1900
iter : 2000
iter : 2100
iter : 2200
iter : 2300
iter : 2400
iter : 2500
iter : 2600
iter : 2700
iter : 2800
iter : 2900
iter : 3000
iter : 3100
iter : 3200
iter : 3300
iter : 3400
iter : 3500
iter : 3600
iter : 3700
iter : 3800
iter : 3900
iter : 4000
iter : 4100
iter : 4200
iter : 4300
iter : 4400
iter : 4500
iter : 4600
iter : 4700
iter : 4800
iter : 4900
iter : 5000
iter : 5100
iter : 5200
iter : 5300
iter : 5400
iter : 5500
iter : 5600
iter : 5700
iter : 5800
iter : 5900
iter : 6000
iter : 6100
iter : 6200
iter : 6300
iter : 6400
iter : 6500
iter : 6600
iter : 6700
iter : 6800
iter : 6900
iter : 7000
iter : 7100
iter : 7200
iter : 7300
iter : 7400
iter : 7500
iter : 7600
iter : 7700
iter : 7800
iter : 7900
iter : 8000
iter : 8100
iter : 8200
iter : 8300
iter : 8400
iter : 8500
iter : 8600
iter : 8700
iter : 8800
iter : 8900
iter : 9000
iter : 9100
iter : 9200
iter : 9300
iter : 9400
iter : 9500
iter : 9600
iter : 9700
iter : 9800
iter : 9900
save done!!
matching_pennies
iter : 0
iter : 1000
iter : 2000
iter : 3000
iter : 4000
iter : 5000
iter : 6000
iter : 7000
iter : 8000
iter : 9000
Traceback (most recent call last):
  File "synchro_world.py", line 136, in <module>
    W.draw_network(f_name=RESULT_NAME+"_fig.png", w_degree=True)
  File "synchro_world.py", line 82, in draw_network
    nx.draw(self.G, pos=nx.circular_layout(self.G), nodelist=d.keys(), width=0.1, alpha=0.5, node_color=color, node_size=[20+v*5 for v in d.values()])
  File "/home/yamazaki/.pyenv/versions/3.5.2/lib/python3.5/site-packages/networkx/drawing/nx_pylab.py", line 114, in draw
    cf = plt.gcf()
  File "/home/yamazaki/.pyenv/versions/3.5.2/lib/python3.5/site-packages/matplotlib/pyplot.py", line 586, in gcf
    return figure()
  File "/home/yamazaki/.pyenv/versions/3.5.2/lib/python3.5/site-packages/matplotlib/pyplot.py", line 535, in figure
    **kwargs)
  File "/home/yamazaki/.pyenv/versions/3.5.2/lib/python3.5/site-packages/matplotlib/backends/backend_tkagg.py", line 84, in new_figure_manager
    return new_figure_manager_given_figure(num, figure)
  File "/home/yamazaki/.pyenv/versions/3.5.2/lib/python3.5/site-packages/matplotlib/backends/backend_tkagg.py", line 92, in new_figure_manager_given_figure
    window = Tk.Tk()
  File "/home/yamazaki/.pyenv/versions/3.5.2/lib/python3.5/tkinter/__init__.py", line 1868, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable
